### ISSUE TITLE
Chemist starter bundles

### ISSUE THREAD

**Issue by Dagdammit** [yogstation13/yogstation#1802]
Give chemists a bluespace voucher in their starting inventory. When used, it prompts them to pick one of the following bundles (a la chaplain null rod), morphing into a box with a 30u bottle of each listed chemical.

Joker: Laughter, Colorful Reagent, Barber's Aid, and Itching Powder

Building Blocks: Oil, Acetone, Diethylamine, and Phenol

Starter Medicines: Kelotane, Bicardine, Antitoxin, Dexalin, and Mannitol

Advanced Medicines: Sterilizine, Oculine, Incausiate, Calomel, Mutadone, and Antihol

Mad Scientist: Omnizine, Holy Water, Unstable Mutagen, Synthflesh, and Stabilizing Reagent

Team Player
-100u Unstable Mutagen in wrapped gift labeled "to Botany"
-100u of Space Cleaner, "to Janitor"
-30u Pentetic Acid and 15u of Mutadone, "to Genetics"
-30u of Unstable Mutagen and Synaptizine, "to Virology"

**Comment by ShadowDeath6** [comment]
no

we're not giving chem more reason to fuck off and validhunt after giving all their shit to people, that just makes the job even more of a joke than it already is

**Comment by Super3222** [comment]
Team Player is bad.
Everything else is good.
We could even put these in the preferences for MUH IMMERSHUNS. Blue space things that fly into your hands are uncanny.

30u of omnizine is meh. It's suppose to be a rare chemical that you dream of unless you're the CMO.

**Comment by Rocks4Free** [comment]
> It's suppose to be a rare chemical that you dream of unless you're the CMO.

Just give botany some mutagen, it's really not that hard

**Comment by Dagdammit** [comment]
Could just make it a box sprite and it still toggles on first use.

Omnizine's in warm donk pockets and the first plant botany mutates.

**Comment by McDonald072** [comment]
Everyone would go for mad scientist, everything else can be made in a few minutes

**Comment by Dagdammit** [comment]
Personally I'd go Building Blocks every time, no question.

**Comment by Super3222** [comment]
It's all about preference.
Mad Scientist has some extremely easy to make chems. My only trigger is omnizine. Seeing it paired up with holy water makes it worse because chemists will create the life reagent without needing to step out of their lab (which sort of defeats the purpose).

They could also spawn with these in a chemistry satchel (ones that collect bottles). Remind me to tweak the 	display_contents_with_number = 1 variable so different bottles don't stack.

**Comment by Dagdammit** [comment]
What would you replace omnizine with?

**Comment by ShadowDeath6** [comment]
replace it with 300u of potassium and water in one beaker

**Comment by Dagdammit** [comment]
"You mean a cryostasis beaker?" "No it should immediately explode."

**Comment by Super3222** [comment]
Cryptobiolin, maybe?

**Comment by McDonald072** [comment]
It takes 3 clicks to mix it.

**Comment by Super3222** [comment]
So does Unstable Mutagen. And Stabilizing Reagent. And most of the chemicals in the starter bundle.

**Comment by McDonald072** [comment]
Yep. So every bundle apart from mad scientist is useless.

**Comment by Dagdammit** [comment]
They aren't supposed to have much of the kind of use that you apparently
care about.

**Comment by 1fbff5f83b23d39d38b1dfcb4cac8d9b** [comment]
But, why would you take anything but the rare chemicals?

**Comment by ShadowDeath6** [comment]
Your line of thought is that it takes more than one sixteenth of the round for chemistry to get everything besides the rare stuff.

**Comment by Dagdammit** [comment]
They're just meant to save a little time + hassle,  and give the novice
chemist a small leg up. I've never had trouble getting omnizine as chemist
(just ask chef for warm donk pockets or make them yourself), so it didn't
seem that big a deal. Maybe reducing it to 10u would work?

Putting it differently: I'm pretty damn sure I could get my hands on
omnizine before any of you could produce colorful reagent.

**Comment by Dagdammit** [comment]
@ShadowDeath6 You and I would never pick the starter medicine bundle sure, but a novice chemist (to whom all recipes feel confusing) would appreciate it. Helps them supply people with healing chems, and once they've learned a little they'll go "hey, I don't need this crutch any more" and pick a different box.

I really just like the idea of people having this choice- it seems like something where everyone could have a different "right" option they always go for. It's not supposed to be any kind of significant chemistry buff, just a way to make life slightly more convenient by taking care of some small bit of the job feels difficult/tedious to that player.

**Comment by ShadowDeath6** [comment]
If you're not intending it be a chemistry buff then I'd remove the bundles that give chemistry the things it doesn't get inside the lab, removing le interdepartmental communication is le no bueno

**Comment by Dagdammit** [comment]
Any suggestions as to alternatives?

I figured small amounts made it fine, you still need other departments to
get any more.

**Comment by ShadowDeath6** [comment]
Personally I don't think any of it is needed considering medical already has access to more than enough medicine to deal with problems.
Giving chem easy stuff to make like mutagen or bicaridine is just spoonfeeding and doesn't really change the learning curve of "make easy stuff for medical first then do other stuff".

**Comment by grubstank** [comment]
I think this is a really bad idea. You can make just about anything in chemistry within a few minutes. Stocking the chem fridge is an extremely simple round start duty. The whole point of chemistry is to make chems to help the other departments. All this does is remove (admittedly grindy) gameplay for chemists without adding gameplay to actually work towards. This will just increase the number of chemists who turn to validhunting out of boredom.

**Comment by Dagdammit** [comment]
It’s an appetizer/sampler pack/training wheels. Meant to save a little
piece of grind for veterans and give a nice little bit of assistance to
newbies, while suggesting various things they can use the job to get up to.
It’s not meant to seriously impact the logistics of what a chemist can do.

Also an old ass suggestion that nobody cared quite enough to make (myself
included), necroing just increases chance it might get done after all. =P

**Comment by grubstank** [comment]
Nah, I'm hoping that these necro'd issues can get closed. There's a whole load of generally-consensus bad or outdated threads that need closing.

**Comment by Dagdammit** [comment]
Well, yeah. They made people post suggestions here for ages, it was
inevitable.
